newhelm/sut_decorator.py

```diff
@@ -43,3 +45,30 @@ def wrapped_init(self, *args, **kwargs):
 def _validate_init_signature(init):
     params = list(inspect.signature(init).parameters.values())
     assert params[1].name == "uid", "All SUTs must have UID as the first parameter."
+
+
+def _override_translate_response(cls: Type[PromptResponseSUT]) -> None:
```

Comment:
I'll add Tests if you all think this idea is worth the potential confusion it may cause.